packages/swapper/src/swappers/ChainflipSwapper/swapperApi/getTradeRate.ts

```diff
@@ -16,5 +16,5 @@ export const getTradeRate = async (
   deps: SwapperDeps,
 ): Promise<Result<TradeRate[], SwapErrorRight>> => {
   const rates = await _getTradeQuote(input as unknown as CommonTradeQuoteInput, deps)
-  return rates as Result<TradeRate[], SwapErrorRight>
+  return rates as unknown as Result<TradeRate[], SwapErrorRight>
```

Comment:
hell yeah 214784c534

Reply:
@gomesalexandre it might be worth instead of duplicating to build an helper like `convertRateToQuote`, as we do for Solana instructions on chain adapter side, so you could easily `map(convertRateToQuote)`

Reply:
Would prefer to avoid it as this would validate the current way, however, the current way is far from optimal and I would prefer to spend time actually improving types to avoid casts, rather than adding bandaid utils 🙏🏽